typify-impl/tests/github.out

```diff
@@ -83834,9 +83834,9 @@ impl ::std::convert::From<&RepositoryUnarchivedRepositoryPermissions>
 #[derive(:: serde :: Deserialize, :: serde :: Serialize, Clone, Debug)]
 #[serde(untagged)]
 pub enum RepositoryUnarchivedRepositoryPushedAt {
-    Variant0(i64),
-    Variant1(::chrono::DateTime<::chrono::offset::Utc>),
-    Variant2,
+    Integer(i64),
+    String(::chrono::DateTime<::chrono::offset::Utc>),
```

Comment:
This is fine, but does suggest room for improvement in a future PR -- perhaps we consider the type name of the generated type as well and end up with variants `I64` / `Integer', `DateTime`, `Null` or we consider the format field as well as the type field when constructing more generic variants names.

Reply:
Decided to use that over the generic type. Seems to have made a few extra improvement such as DateTime and Uuid!